logging/config/cluster-logging-collector.adoc

```diff
@@ -14,7 +14,7 @@ You can configure log rotation, log location, use an external log aggregator, an
 You must set cluster logging to Unmanaged state before performing these configurations, unless otherwise noted.
 For more information, see xref:../../logging/config/cluster-logging-management.adoc#cluster-logging-management[Changing the cluster logging management state].
 
-Operators in an unmanaged state are unsupported and the cluster administrator assumes full control of the individual component configurations and upgrades.
+Operators in an modules/cluster-logging-collector-chunk.adocunmanaged state are unsupported and the cluster administrator assumes full control of the individual component configurations and upgrades.
```

Comment:
Looks like this slid in accidentally, it's building as "...an modules/cluster-logging-collector-chunk.adocunmanaged state are..."